primitives/bytes/b4_test.go

```diff
@@ -386,3 +275,159 @@ func TestBytes4HashTreeRoot(t *testing.T) {
 		})
 	}
 }
+
+func TestBytes4Equals(t *testing.T) {
```

Comment:
I think this is more than we need.
There could be a single UT in `version` package where we list the versions in a slice and we verify that each element is strictly before the next one

Reply:
done